Lib/test/test_capi/test_unicode.py

```diff
@@ -11,6 +11,53 @@
 
 class CAPITest(unittest.TestCase):
 
+    @support.cpython_only
```

Comment:
What do you think of moving these tests to test_codecs? For me, these APIs are more related to codecs than "Unicode". test_codecs already has multiple tests about the UTF-8 encoding.

Reply:
Done. But would not be more convenient to have tests for all `PyUnicode_*` functions in a single file?